packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -47,6 +49,7 @@ export interface ViewEvent {
   loadingTime?: Duration
   loadingType: ViewLoadingType
   cumulativeLayoutShift?: number
+  scrollMetrics?: ScrollMetrics
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏scrollMetrics is always defined

```suggestion
  scrollMetrics: ScrollMetrics
```